examples/README.rst

```diff
@@ -0,0 +1,156 @@
+libCEED: Examples
+==================================================
+This page provides a brief description of the examples for the libCEED
+library.
+
+
+Basic libCEED Examples
+--------------------------------------------------
+
+Two examples are provided that rely only upon libCEED without any
+external libraries.
+
+
+Example 1: ex1-volume
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example uses the mass matrix to compute the length, area, or volume
+of a region, depending upon runtime parameters.
+
+
+Example 2: ex2-surface
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example uses the diffusion matrix to compute the surface area of a
+region, depending upon runtime parameters.
+
+
+Bakeoff Problems
+--------------------------------------------------
+
+This section provides a brief description of the bakeoff problems, used
+as examples for the libCEED library. These bakeoff problems are
+high-order benchmarks designed to test and compare the performance of
+high-order finite element codes.
+
+For further documentation, readers may wish to consult the `CEED
+documentation <http://ceed.exascaleproject.org/bps/>`__ of the bakeoff
+problems.
+
+
+Bakeoff Problem 1
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Bakeoff problem 1 is the *L2* projection problem into the finite element
+space.
+
+The supplied examples solve *B u = f*, where *B* is the mass matrix.
```

Comment:
Why not use `:math:`, or is this never rendered as part of the Sphinx site? I'm a little concerned about the same information appearing in too many different places, because that'll inevitably lead to inconsistencies.

Reply:
Thanks. This was converted directly (using pandoc this time!) from the `examples/README.md` so I guess that was the way originally math was redered for the markdown file. If we prefer use the same content as in the [sphinx bps page](https://libceed.readthedocs.io/en/latest/examples/bps.html) I can go ahead and replace this content and then use the `include` directive again to avoid duplication of content. But the way I described the BPs in the sphinx page is a bit more _mathy_ than what it was originally in this file and the CEED website.

Reply:
I liked that strategy. Different descriptions of the same thing have a tendency to become inconsistent and I don't see a high strategic value of having them separate.

Reply:
Okay. I've done inclusion of the parts that would render in a simple rst viewer, like on GitHub. I couldn't include complex math statements, so I included what I could.